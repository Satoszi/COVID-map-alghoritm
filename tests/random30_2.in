30 30 7 140
random30_2
+|++|.+||+.+..++.+-+++..+.++++
..+|.-+-++|+-+.++..+++|.+.-|.-
+...|+.....++.||+|-.-+.-.+|-..
+++..+-++|..++.-+.||.-|.+.+++.
-..-++++...++-++.+.++..+.+....
.++++++.|..+.--.-.|+|+++..|+++
++++.+-|..|..-|.+.+...++||++++
+|+.||.-+++.+||.|+.|++++++-..+
+.|++-+.-+++.+.++++.+...++..+.
++...|.++.....|-++-++..++.|++-
+++++-.|+|+.|++-+..+-++..|.|++
.+..+.++....+...+++..++|+|.++.
+.||-+++.+.+....+++.|.++.|...|
+|--.+++.-+++-+||++-..++++|++.
.+.--..+..-+--+|++.+|++|+++-++
....+.....|+..+-++++-+-+..|+-+
+-.+++.-+-+..+++.-+.+.|+....+-
+.-.+.+++.++++.+..+.||+.|.++|+
.+.-++.+..++-.+...++.++|.+.++.
.+.+.+++-.+++...+..|-+.+.+++|+
.++++++....-...|+...+..-|+.+..
---+.-|+++.+.+|..+.+-.++|++-..
+...++-++.+|+.+-+.|-.+++..+.+.
++.++..+++++.....+.+.+|-+-++-|
++|++-.-.+++.+|-.++++-.+.....|
.++..|.+++.+-+++|.-++.-+.+-+..
....+...+|.++.+-++-+.+..|++..|
.|++-.++++.+.-+|..+++.|++.+.++
||-+.++++++-++.+.--++.++++||++
|..+..||.-..+++-+|.....|+-++-+
