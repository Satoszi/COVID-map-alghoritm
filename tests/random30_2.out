17 29
16 29
9 29
17 28
0 28
15 27
2 27
1 27
29 26
24 26
21 26
4 26
27 25
7 25
5 25
23 24
15 24
14 24
7 24
3 24
29 23
23 23
17 23
28 22
19 22
18 22
12 22
0 22
22 21
19 21
17 21
14 21
13 21
11 21
9 21
6 21
5 21
0 21
24 20
23 20
20 20
15 20
11 20
29 19
19 19
16 19
12 19
25 18
23 18
14 18
1 18
29 17
24 17
15 17
2 17
29 16
22 16
20 16
7 16
0 16
27 15
22 15
4 15
20 14
3 14
19 13
15 13
2 13
1 13
29 12
20 12
11 12
9 12
4 12
3 12
2 12
0 12
24 11
25 10
21 10
12 10
10 10
9 10
8 10
5 10
26 9
23 9
14 9
5 9
28 8
19 7
13 7
12 7
4 7
0 7
14 6
13 6
10 6
7 6
26 5
21 5
18 5
16 5
13 5
8 5
25 4
17 4
14 4
0 4
24 3
22 3
21 3
19 3
18 3
9 3
27 2
26 2
25 2
23 2
18 2
17 2
14 2
5 2
4 2
0 2
29 1
26 1
22 1
13 1
10 1
26 0
24 0
17 0
14 0
7 0
6 0
4 0
2 0
1 0
0 0
